### PR TITLE
More correct handling of finished download

### PR DESCRIPTION
**Pull request by indivisible**
Fixes ElPumpo/TinyNvidiaUpdateChecker#26

### CODE REVIEW

**Comment by ElPumpo** [comment]
Isn't the following smarter?
```
webClient.DownloadFileCompleted += delegate (object sender, AsyncCompletedEventArgs e)
{
    if (e.Cancelled || e.Error != null) {
        File.Delete(savePath + driverFileName);
    } else {
        notifier.Set();
    }
};
```

**Comment by indivisible** [comment]
It is.

File: TinyNvidiaUpdateChecker/MainConsole.cs [hunk 1]
```diff
@@ -812,14 +812,14 @@ private static void DownloadDriver()
                             webClient.DownloadProgressChanged += delegate (object sender, DownloadProgressChangedEventArgs e)
                             {
                                 progress.Report((double)e.ProgressPercentage / 100);
-
-                                if (e.BytesReceived >= e.TotalBytesToReceive) notifier.Set();
                             };
 
                             webClient.DownloadFileCompleted += delegate (object sender, AsyncCompletedEventArgs e)
                             {
                                 if(e.Cancelled) {
```
Comment (ElPumpo) [inline TinyNvidiaUpdateChecker/MainConsole.cs]: Also use the information `e` provides: `if (e.Cancelled || e.Error != null) {`

File: TinyNvidiaUpdateChecker/MainConsole.cs [hunk 2]
```diff
@@ -812,14 +812,14 @@ private static void DownloadDriver()
...
                                     File.Delete(savePath + driverFileName);
+                                }else{
```
Comment (ElPumpo) [inline TinyNvidiaUpdateChecker/MainConsole.cs]: Please try to follow how my code looks, use spaces between the brackets.

**Pull request closed without merging**